__tests__/commands/init.js

```diff
@@ -15,6 +15,9 @@ const execInit = buildRun.bind(
   ConsoleReporter,
   fixturesLoc,
   (args, flags, config, reporter, lockfile): Promise<void> => {
+    if (flags.hasOwnProperty('questionMap')) {
```

Comment:
Not a fan of this change but I'm okay with it until I can find a better solution.